msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/InteractiveRequest.java

```diff
@@ -55,24 +56,31 @@ URL authorizationUrl() {
     }
 
     private void validateRedirectUrl(URI redirectUri) {
+        String host = redirectUri.getHost();
+        String scheme = redirectUri.getScheme();
+        InetAddress address;
+
+        //Validate URI scheme. Only http is valid, as determined by the HttpListener created in AcquireTokenByInteractiveFlowSupplier.startHttpListener()
+        if (scheme == null || !scheme.equals("http")) {
+            throw new MsalClientException(String.format(
+                    "Only http is supported for the redirect URI of an interactive request, but \"%s\" was found. For more information about redirect URI formats, see https://aka.ms/msal4j-interactive-request", scheme),
```

Comment:
nit: When WAM is more stable, this aka.ms link should point to it. A WAM powered app would still need this redirect uri for when the broker is not present (old Windows, Mac, Linux)